app/views/publishers/vacancies/job_applications/_banner.html.slim

```diff
@@ -0,0 +1,41 @@
+= render BannerComponent.new do
```

Comment:
if you put `= render BannerComponent.new do classes: "govuk-!-padding-bottom-0" do` fixes the padding thing i mentioned below